README.md

```diff
@@ -0,0 +1,12 @@
+# Lab 16 - AWS: Cloud Servers
+
+## Deployed Servers
+
+[deployed server 1](http://expressserver-env-1.eba-ehmp3vas.us-west-2.elasticbeanstalk.com/)
+
+[deployed server 2](http://expressserver-env-2.eba-ehmp3vas.us-west-2.elasticbeanstalk.com/)
+
+## Process
+
+The deployment did not succeed when I tried to upload my own code of a basic express server. I found a sample file in AWS documentation and tried deploying that, which was successful. I was not able to figure out how to troubleshoot the deployment that did not work. I will need to ask for asistance from instructors or TA's to figure out what went wrong.
+
```

Comment:
Once you overcome the troubleshooting part, I highly advise you to rewrite this part and put the steps on how to deploy to  AWS: EC2. If you want, you can add a troubleshooting section explaining how you overcame the issues. So if you do deploy again, you can look at this readme as a starting point